config/locales/views.de.yml

```diff
@@ -17,6 +17,7 @@ de:
     until: Bis
     no_list_entries: Keine Einträge gefunden.
     confirm_delete: Wollen Sie diesen Eintrag wirklich löschen?
+    confirm_delete_person: Wollen Sie %{person} wirklich löschen?
```

Comment:
Controller/View spezifische Keys bitte auch in den entsprechenden Namespaces ablegen, und nicht global. Bisheriges Naming Schema beachten.